core/components/molecules/modalFooter/ModalFooter.tsx

```diff
@@ -5,23 +5,28 @@ import { BaseProps, extractBaseProps } from '@/utils/types';
 export interface ModalFooterProps extends BaseProps {
   open?: boolean;
   children: React.ReactNode;
+  stickToBottom: boolean;
+  seperator?: boolean;
+  inSidesheet?: boolean;
 }
 
 export const ModalFooter = (props: ModalFooterProps) => {
-  const { open, children, className } = props;
+  const { open, children, className, stickToBottom, seperator, inSidesheet } = props;
   const baseProps = extractBaseProps(props);
 
   const classes = classNames({
-    'Modal-footer': true
+    'Modal-footer': true,
+    ['Modal-footer--inModal']: !inSidesheet,
```

Comment:
Why not name it `Modal-footer--inSidesheet` and apply it in case of `inSidesheet: true`?